### PR TITLE
Refactor types in Angular Demos p.2

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts [hunk 1]
```diff
@@ -9,8 +9,12 @@ import {
   DxTagBoxModule,
   DxFilterBuilderModule,
 } from 'devextreme-angular';
+import { DxFilterBuilderTypes } from 'devextreme-angular/ui/filter-builder';
+import { DxFilterBuilder } from 'devextreme-vue';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts]: Why do you use import from vue?

File: JSDemos/Demos/PivotGrid/ChartIntegration/Angular/app/app.component.html [hunk 2]
```diff
@@ -14,7 +14,6 @@
     [allowSortingBySummary]="true"
     [allowFiltering]="true"
     [showBorders]="true"
-    [allowSortingBySummary]="true"
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/PivotGrid/ChartIntegration/Angular/app/app.component.html:17]: Why did you remove the prop?

Comment (GoodDayForSurf) [reply JSDemos/Demos/PivotGrid/ChartIntegration/Angular/app/app.component.html:17]: look upper, it's double

File: JSDemos/Demos/FileUploader/FileUploading/Angular/app/app.component.ts [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+import {
+  NgModule, Component, enableProdMode, Pipe, PipeTransform,
+} from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
-import { DxCheckBoxModule, DxFileUploaderModule, DxSelectBoxModule } from 'devextreme-angular';
+import {
+  DxCheckBoxModule, DxFileUploaderModule, DxSelectBoxModule,
+} from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/FileUploader/FileUploading/Angular/app/app.component.ts:9]:
```suggestion
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
import { DxFileUploaderModule } from 'devextreme-angular/ui/file-uploader';
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```

File: JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts [hunk 4]
```diff
@@ -1,16 +1,12 @@
...
+import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxListModule,
-  DxButtonModule,
-  DxTagBoxModule,
-  DxFilterBuilderModule,
-} from 'devextreme-angular';
+import { DxListModule, DxButtonModule, DxTagBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts:4]:
```suggestion
import { DxListModule } from 'devextreme-angular/ui/list';
import { DxButtonModule } from 'devextreme-angular/ui/button';
import { DxTagBoxModule } from 'devextreme-angular/ui/tag-box';
```

File: JSDemos/Demos/FloatingActionButton/Overview/Angular/app/app.component.ts [hunk 5]
```diff
@@ -3,16 +3,11 @@ import {
...
 import config from 'devextreme/core/config';
 import repaintFloatingActionButton from 'devextreme/ui/speed_dial_action/repaint_floating_action_button';
-import {
-  DxDataGridModule,
-  DxDataGridComponent,
-  DxSpeedDialActionModule,
-  DxSelectBoxModule,
-} from 'devextreme-angular';
-
+import { DxSpeedDialActionModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/FloatingActionButton/Overview/Angular/app/app.component.ts:8]:
```suggestion
import { DxSpeedDialActionModule } from 'devextreme-angular/ui/speed-dial-action';
```

File: JSDemos/Demos/Form/ColumnsAdaptability/Angular/app/app.component.ts [hunk 6]
```diff
@@ -1,10 +1,8 @@
-import {
-  NgModule, Component, ViewChild, enableProdMode,
-} from '@angular/core';
+import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxFormModule, DxCheckBoxModule } from 'devextreme-angular';
-
+import { DxFormModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Form/ColumnsAdaptability/Angular/app/app.component.ts:4]:
```suggestion
import { DxFormModule } from 'devextreme-angular/ui/form';
```

File: JSDemos/Demos/Form/Overview/Angular/app/app.component.ts [hunk 7]
```diff
@@ -1,13 +1,8 @@
...
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxCheckBoxModule,
-  DxSelectBoxModule,
-  DxNumberBoxModule,
-  DxFormModule,
-} from 'devextreme-angular';
-
+import { DxCheckBoxModule, DxSelectBoxModule, DxNumberBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Form/Overview/Angular/app/app.component.ts]:
```suggestion
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
import { DxNumberBoxModule } from 'devextreme-angular/ui/number-box';
```

File: JSDemos/Demos/Form/UpdateItemsDynamically/Angular/app/app.component.ts [hunk 8]
```diff
@@ -1,12 +1,10 @@
...
-  NgModule, Component, enableProdMode, AfterViewInit,
-} from '@angular/core';
+import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxFormModule,
-} from 'devextreme-angular';
-
+import { DxFormModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Form/UpdateItemsDynamically/Angular/app/app.component.ts:4]:
```suggestion
import { DxFormModule } from 'devextreme-angular/ui/form';
```

File: JSDemos/Demos/Gallery/Overview/Angular/app/app.component.ts [hunk 9]
```diff
@@ -1,8 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxCheckBoxModule, DxGalleryModule } from 'devextreme-angular';
-
+import { DxGalleryModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Gallery/Overview/Angular/app/app.component.ts:4]:
```suggestion
import { DxGalleryModule } from 'devextreme-angular/ui/gallery';
```

File: JSDemos/Demos/Gantt/ChartAppearance/Angular/app/app.component.ts [hunk 10]
```diff
@@ -2,9 +2,8 @@ import { ViewEncapsulation } from '@angular/compiler/src/core';
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxGanttModule, DxCheckBoxModule, DxSelectBoxModule, DxDateBoxModule,
-} from 'devextreme-angular';
+import { DxCheckBoxModule, DxSelectBoxModule, DxDateBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Gantt/ChartAppearance/Angular/app/app.component.ts:5]:
```suggestion
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
import { DxDateBoxModule } from 'devextreme-angular/ui/date-box';
```

File: JSDemos/Demos/Gauges/VariableNumberOfSubvalueIndicators/Angular/app/app.component.ts [hunk 11]
```diff
@@ -1,7 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxLinearGaugeModule, DxSelectBoxModule } from 'devextreme-angular';
+import { DxSelectBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Gauges/VariableNumberOfSubvalueIndicators/Angular/app/app.component.ts:4]:
```suggestion
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```

File: JSDemos/Demos/HtmlEditor/ToolbarCustomization/Angular/app/app.component.ts [hunk 12]
```diff
@@ -1,15 +1,10 @@
...
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxHtmlEditorComponent,
-  DxHtmlEditorModule,
-  DxPopupComponent,
-  DxPopupModule,
-} from 'devextreme-angular';
-
+import { DxHtmlEditorModule, DxPopupModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/HtmlEditor/ToolbarCustomization/Angular/app/app.component.ts:6]:
```suggestion
import { DxHtmlEditorModule } from 'devextreme-angular/ui/html-editor';
import { DxPopupModule } from 'devextreme-angular/ui/popup';
```

File: JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts [hunk 13]
```diff
@@ -1,14 +1,16 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxListModule } from 'devextreme-angular';
-
+import { DxListComponent, DxListModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts]:
```suggestion
import { DxListComponent, DxListModule } from 'devextreme-angular/ui/list';
```

File: JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts [hunk 14]
```diff
@@ -1,14 +1,16 @@
...
 import { Task, Service } from './app.service';
 
 if (!/localhost/.test(document.location.host)) {
   enableProdMode();
 }
 
+type ListItemDragging = DxListComponent['itemDragging'];
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts]: I am not sure about this type. In this case I would prefer not to use types for handlers

File: JSDemos/Demos/Lookup/EventHandling/Angular/app/app.component.ts [hunk 15]
```diff
@@ -1,8 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxLookupModule, DxSelectBoxModule } from 'devextreme-angular';
-
+import { DxSelectBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Lookup/EventHandling/Angular/app/app.component.ts:4]:
```suggestion
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```

File: JSDemos/Demos/Map/Routes/Angular/app/app.component.ts [hunk 16]
```diff
@@ -1,13 +1,10 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxMapModule, DxSelectBoxModule } from 'devextreme-angular';
-
+import { DxMapModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Map/Routes/Angular/app/app.component.ts:4]:
```suggestion
import { DxMapModule } from 'devextreme-angular/ui/map';
```

File: JSDemos/Demos/Menu/Overview/Angular/app/app.component.ts [hunk 17]
```diff
@@ -1,9 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-
-import { DxMenuModule, DxSelectBoxModule, DxCheckBoxModule } from 'devextreme-angular';
-
+import { DxSelectBoxModule, DxCheckBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Menu/Overview/Angular/app/app.component.ts:4]:
```suggestion
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
```

File: JSDemos/Demos/PivotGrid/ExcelJsHeaderAndFooter/Angular/app/app.component.ts [hunk 18]
```diff
@@ -1,12 +1,13 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxCheckBoxModule } from 'devextreme-angular';
-
+import { DxCheckBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/PivotGrid/ExcelJsHeaderAndFooter/Angular/app/app.component.ts:4]:
```suggestion
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
```

File: JSDemos/Demos/PivotGrid/FieldPanel/Angular/app/app.component.ts [hunk 19]
```diff
@@ -1,8 +1,9 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxCheckBoxModule } from 'devextreme-angular';
+import { DxCheckBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/PivotGrid/FieldPanel/Angular/app/app.component.ts:4]:
```suggestion
import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
```

File: JSDemos/Demos/PivotGrid/IntegratedFieldChooser/Angular/app/app.component.ts [hunk 20]
```diff
@@ -1,9 +1,11 @@
 import {
-  NgModule, Component, ViewChild, enableProdMode,
+  NgModule, Component, enableProdMode,
 } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxSelectBoxModule } from 'devextreme-angular';
+import { DxSelectBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/PivotGrid/IntegratedFieldChooser/Angular/app/app.component.ts:4]:
```suggestion
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```

File: JSDemos/Demos/PivotGrid/StatePersistence/Angular/app/app.component.ts [hunk 21]
```diff
@@ -1,8 +1,9 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxButtonModule } from 'devextreme-angular';
+import { DxButtonModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/PivotGrid/StatePersistence/Angular/app/app.component.ts]:
```suggestion
import { DxButtonModule } from 'devextreme-angular/ui/button';
```

File: JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/Angular/app/app.component.ts [hunk 22]
```diff
@@ -2,7 +2,8 @@ import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { DatePipe } from '@angular/common';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxRangeSelectorModule, DxSelectBoxModule } from 'devextreme-angular';
+import { DxSelectBoxModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/Angular/app/app.component.ts:5]:
```suggestion
import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```

File: JSDemos/Demos/RangeSelector/UseRangeSelectionForFiltering/Angular/app/app.component.ts [hunk 23]
```diff
@@ -1,8 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxRangeSelectorModule, DxDataGridModule } from 'devextreme-angular';
-
+import { DxDataGridModule } from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/RangeSelector/UseRangeSelectionForFiltering/Angular/app/app.component.ts:4]:
```suggestion
import { DxDataGridModule } from 'devextreme-angular/ui/data-grid';
```

**Review by dxvladislavvolkov — CHANGES_REQUESTED** [review]

File: JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts [hunk 24]
```diff
@@ -1,16 +1,14 @@
...
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxListModule,
-  DxButtonModule,
-  DxTagBoxModule,
-  DxFilterBuilderModule,
-} from 'devextreme-angular';
+import { DxListModule } from 'devextreme-angular/ui/list';
+import { DxButtonModule } from 'devextreme-angular/ui/button';
+import { DxTagBoxModule } from 'devextreme-angular/ui/tag-box';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/FilterBuilder/Customization/Angular/app/app.component.ts]: Revert. Only Modules from root.

File: JSDemos/Demos/FloatingActionButton/Overview/Angular/app/app.component.ts [hunk 25]
```diff
@@ -3,16 +3,11 @@ import {
...
 import config from 'devextreme/core/config';
 import repaintFloatingActionButton from 'devextreme/ui/speed_dial_action/repaint_floating_action_button';
-import {
-  DxDataGridModule,
-  DxDataGridComponent,
-  DxSpeedDialActionModule,
-  DxSelectBoxModule,
-} from 'devextreme-angular';
-
+import { DxSpeedDialActionModule } from 'devextreme-angular/ui/speed-dial-action';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/FloatingActionButton/Overview/Angular/app/app.component.ts]: Module from root.

File: JSDemos/Demos/Form/ColumnsAdaptability/Angular/app/app.component.ts [hunk 26]
```diff
@@ -1,10 +1,8 @@
-import {
-  NgModule, Component, ViewChild, enableProdMode,
-} from '@angular/core';
+import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxFormModule, DxCheckBoxModule } from 'devextreme-angular';
-
+import { DxFormModule } from 'devextreme-angular/ui/form';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/Form/ColumnsAdaptability/Angular/app/app.component.ts]: root

File: JSDemos/Demos/Form/Overview/Angular/app/app.component.ts [hunk 27]
```diff
@@ -1,13 +1,10 @@
...
-import {
-  DxCheckBoxModule,
-  DxSelectBoxModule,
-  DxNumberBoxModule,
-  DxFormModule,
-} from 'devextreme-angular';
-
+import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
+import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
+import { DxNumberBoxModule } from 'devextreme-angular/ui/number-box';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/Form/Overview/Angular/app/app.component.ts]: Revert

File: JSDemos/Demos/Form/UpdateItemsDynamically/Angular/app/app.component.ts [hunk 28]
```diff
@@ -1,12 +1,10 @@
...
-  NgModule, Component, enableProdMode, AfterViewInit,
-} from '@angular/core';
+import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import {
-  DxFormModule,
-} from 'devextreme-angular';
-
+import { DxFormModule } from 'devextreme-angular/ui/form';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/Form/UpdateItemsDynamically/Angular/app/app.component.ts]: Revert

File: JSDemos/Demos/Gallery/Overview/Angular/app/app.component.ts [hunk 29]
```diff
@@ -1,8 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxCheckBoxModule, DxGalleryModule } from 'devextreme-angular';
-
+import { DxGalleryModule } from 'devextreme-angular/ui/gallery';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/Gallery/Overview/Angular/app/app.component.ts]: root

File: JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts [hunk 30]
```diff
@@ -1,14 +1,16 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxListModule } from 'devextreme-angular';
-
+import { DxListModule } from 'devextreme-angular/ui/list';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/List/ItemDragging/Angular/app/app.component.ts]: Revert

File: JSDemos/Demos/List/ListSelection/Angular/app/app.component.ts [hunk 31]
```diff
@@ -4,8 +4,9 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { DxSelectBoxModule, DxListModule, DxCheckBoxModule } from 'devextreme-angular';
 import DataSource from 'devextreme/data/data_source';
 import ArrayStore from 'devextreme/data/array_store';
-
-import { Task, Service } from './app.service';
+import { SingleMultipleAllOrNone } from 'devextreme/common';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/List/ListSelection/Angular/app/app.component.ts]: devextreme-angular/common

File: JSDemos/Demos/PivotGrid/FieldPanel/Angular/app/app.component.ts [hunk 32]
```diff
@@ -1,8 +1,9 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxCheckBoxModule } from 'devextreme-angular';
+import { DxCheckBoxModule } from 'devextreme-angular/ui/check-box';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/PivotGrid/FieldPanel/Angular/app/app.component.ts]: root

File: JSDemos/Demos/PivotGrid/IntegratedFieldChooser/Angular/app/app.component.ts [hunk 33]
```diff
@@ -1,9 +1,11 @@
 import {
-  NgModule, Component, ViewChild, enableProdMode,
+  NgModule, Component, enableProdMode,
 } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxSelectBoxModule } from 'devextreme-angular';
+import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/PivotGrid/IntegratedFieldChooser/Angular/app/app.component.ts]: root

File: JSDemos/Demos/PivotGrid/StatePersistence/Angular/app/app.component.ts [hunk 34]
```diff
@@ -1,8 +1,9 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxPivotGridModule, DxButtonModule } from 'devextreme-angular';
+import { DxButtonModule } from 'devextreme-angular/ui/button';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/PivotGrid/StatePersistence/Angular/app/app.component.ts]: root

File: JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/Angular/app/app.component.ts [hunk 35]
```diff
@@ -2,7 +2,8 @@ import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { DatePipe } from '@angular/common';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxRangeSelectorModule, DxSelectBoxModule } from 'devextreme-angular';
+import { DxSelectBoxModule } from 'devextreme-angular/ui/select-box';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/Angular/app/app.component.ts]: root

File: JSDemos/Demos/RangeSelector/UseRangeSelectionForFiltering/Angular/app/app.component.ts [hunk 36]
```diff
@@ -1,8 +1,8 @@
 import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { DxRangeSelectorModule, DxDataGridModule } from 'devextreme-angular';
-
+import { DxDataGridModule } from 'devextreme-angular/ui/data-grid';
```
Comment (mpreyskurantov) [inline JSDemos/Demos/RangeSelector/UseRangeSelectionForFiltering/Angular/app/app.component.ts]: root

**Review by mpreyskurantov — DISMISSED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**